public/app/core/components/Login/LoginServiceButtons.tsx

```diff
@@ -128,7 +131,7 @@ export const LoginServiceButtons = () => {
     return (
       <a
         key={key}
-        className={`${styles.button} btn btn-medium btn-service btn-service--${service.className || key}`}
+        className={`btn btn-medium btn-service btn-service--${service.className || key} ${styles.button}`}
```

Comment:
Should we use `cx` here to concat class names?

Reply:
I think keeping the classnames until they are migrated to Emotion makes sense.

Reply:
What I mean is can't you do smth like 
```
className={cx(`btn btn-medium btn-service btn-service--${service.className || key}`, styles.button}`)}
```

Reply:
Ah, fair enough